reminder.sh

```
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Cycled health reminder script
+suggestions=(
+  "🧠  Look far away for 20 seconds (20-20-20)"
+  "🦵  Seated or standing leg lifts"
+  "🧘  Gentle neck rolls or head tilt"
+)
+
+index=0
+
+<<<<<<< HEAD
```

Shellcheck (0.10.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)